test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -662,6 +662,15 @@ public static String randomRealisticUnicodeOfCodepointLength(int codePoints) {
         return generateRandomStringArray(maxArraySize, maxStringSize, allowNull, true);
     }
 
+    public static List<String> randomStringList(int minListSize, int maxListSize, int minStringLength, int maxStringLength) {
```

Comment:
Can you add javadocs for this please?

Reply:
Also, do we use the `minStringLength` and `maxStringLength`? Just wondering if instead we could do `Arrays.asList(generateRandomStringArray(...))`, unless we need the unicode-ness and the min/max string size

Reply:
Yeah, I removed the new method and used `generateRandomStringArray`.